### DISCUSSION TITLE
Adding an option to change the font size in the minimap.

### DISCUSSION THREAD

**Discussion by phpjunkie420** [Nasller/CodeGlancePro#106] [Ideas]
Can you add an option for changing the font size? I'd like to be able to make the text bigger in the minimap.

**Comment by Nasller** [comment]
If it is to modify pixels, there are only four levels at present, and the modification cannot be supported yet.

**Reply by phpjunkie420** [reply]
I see, it is the pixels per line. Can you increase the number of pixels up to like 10?

**Reply by Nasller** [reply]
sorry, it's not working yet. are you using a screen with a high scaling ratio?

**Reply by phpjunkie420** [reply]
Yes, my resolution is 3840x1600.

**Reply by phpjunkie420** [reply]
I'd like to blow up the scrollbar to an insane size, and I'd like to see the code on the scrollbar.

**Reply by Nasller** [reply]
I will consider supporting hidpi images in the future
